flixel/animation/FlxAnimationController.hx

```diff
@@ -777,6 +777,55 @@ class FlxAnimationController implements IFlxDestroyable
 		play(AnimName);
 		return AnimName;
 	}
+		
+	public function animationList():Array<FlxAnimation>
+	{
+		var arrayNames:Array<FlxAnimation> = [];
+		
+		for (i in _animations)
+		{
+			arrayNames.push(i);
+		}
+
+		return arrayNames;
+	}
+
+	public function nameList():Array<String>
+	{
+		var anims = animationList();
+		var names:Array<String> = [];
+		for (i in 0...anims.length)
+		{
+			names.push(anims[i].name);
+		}
+
+		return names;
+	}
+
+	public function exists(name:String):Bool
```

Comment:
needs documentaion

Reply:
added the documentation, at least is what I can give as my perspective lol.
Resolved